app/assets/stylesheets/application.css.scss

```diff
@@ -28,3 +28,18 @@
 .downvote {
   color: red;
 }
+
+#foto, #foto img {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line